taskmanager/spec/Application/Task/TaskSpec.php

```diff
@@ -32,4 +32,36 @@ function it_should_has_status()
     {
         $this->getStatus()->shouldReturnAnInstanceOf(Status::class);
     }
+
+    function it_should_be_possible_to_change_status_to_todo()
```

Comment:
@mateuszmaslanka as this is for code modeling and not testing do we need all of them? or just one example transition?